Example/ConsentViewController_ExampleTests/SPConsentManagerSpec.swift

```diff
@@ -51,5 +51,26 @@ class SPConsentManagerSpec: QuickSpec {
                 expect(manager.selectPrivacyManagerId(fallbackId: "1", groupPmId: nil, childPmId: nil))=="1"
             }
         }
+
+        describe("buildPrivacyManagerUrl") {
+            describe("gdpr") {
+                it("build URL with the right parameters") {
+                    var manager = SPConsentManager(accountId: 1, propertyId: 1, propertyName: coordinator.propertyName, campaigns: coordinator.campaigns, delegate: nil)
+                    manager.messageLanguage = .Spanish
+                    var pmUrl = manager.buildGDPRPmUrl(usedId: "1")
+
+                    expect(pmUrl?.absoluteString)=="https://cdn.privacy-mgmt.com/privacy-manager/index.html?consentLanguage=ES&consentUUID&idfaStatus=accepted&message_id=1&pmTab=&site_id=1"
+                }
+            }
+            describe("ccpa") {
+                it("build URL with the right parameters") {
+                    var manager = SPConsentManager(accountId: 1, propertyId: 1, propertyName: coordinator.propertyName, campaigns: coordinator.campaigns, delegate: nil)
+                    manager.messageLanguage = .Spanish
+                    var pmUrl = manager.buildCCPAPmUrl(usedId: "1")
+
+                    expect(pmUrl?.absoluteString)=="https://cdn.privacy-mgmt.com/ccpa_pm/index.html?consentLanguage=ES&consentUUID&idfaStatus=accepted&message_id=1&pmTab=&site_id=1"
```

Comment:
@andresilveirah my local SPConsentManager sets `idfaStatus=accepted` but GitHub says `unknown`. 
I could hardcode but I will make a public-getter property. 
Either way, it is a lame approach but no idea how to do it better with existing `SPConsent` architecture.

Reply:
I think your approach is valid. [There are ways to reset the IDFA status before the test](https://github.com/SourcePointUSA/ios-cmp-app/blob/9fc5b7ae5064fe168ab7dc0177c7f9d0924c1569/Example/SPGDPRExampleAppUITests/ExampleApp.swift#L27), but it's not available before iOS 15.